src/addons/transitions/__tests__/ReactTransitionChildMapping-test.js

```diff
@@ -109,6 +109,7 @@ describe('ReactTransitionChildMapping', function() {
     });
   });
 
+  /*eslint-disable no-undef-init */
```

Comment:
I think we should turn it off, but only for tests. Explicitly writing `var next = undefined` makes the code more readable. That requires making our grunt rule a bit smarter so it can pattern match `__tests__` directories, which should be another commit and codemod.

Let me know if you'd prefer I turn it off globally (for normal code and tests) and I'll do that instead.